docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,57 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
+  [structural type](structural-data-types) that contains `NULL` elements. ({issue}`16140`)
+
+## CLI
+
+* Add `AUTO` output format which switches from `ALIGNED` to `VERTICAL` if
+  the output doesn't fit the current terminal. ({issue}`12208`)
+
+## BigQuery connector
+
+* Add support for writing `DECIMAL` types to BigQuery. ({issue}`16145`)
+
+## Delta Lake connector
+
+* Add support for creating tables with the `change_data_feed_enabled` table
+  property. ({issue}`16129`)
+* Improve query performance on tables that Trino has written to with `INSERT`. ({issue}`16026`)
+* Retry dropping Delta tables registered on AWS Glue to avoid failures due to
+  concurrent modifications. ({issue}`13199`)
+* Fix inaccurate change data feed entries for `MERGE` queries. ({issue}`16127`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Hive connector
+
+* Retry dropping Delta tables registered on AWS Glue to avoid failures due to
+  concurrent modifications. ({issue}`13199`)
+* Remove support for the deprecated `hive-hadoop2` connector name. ({issue}`16166`)
```

Comment:
Should we move this up to be the first item since its kinda a breaking change @martint ?

Reply:
Yes, that's reasonable. We may want to mention what's the expected name to use instead.